pkg/api/wrappers/deployment_install_client.go

```diff
@@ -118,3 +120,13 @@ func (c *InstallStrategyDeploymentClientForNamespace) FindAnyDeploymentsMatching
 	}
 	return deployments, nil
 }
+
+func (c *InstallStrategyDeploymentClientForNamespace) FindAnyDeploymentsMatchingLabels(label labels.Selector) ([]*appsv1.Deployment, error) {
+	deployments, err := c.opLister.AppsV1().DeploymentLister().Deployments(c.Namespace).List(label)
+	// Any errors other than !exists are propagated up
+	if !apierrors.IsNotFound(err) {
```

Comment:
nit: This works, but I think it would be more conventional to do something like this:
```go
if err != nil && !apierrors.IsNotFound(err) {
   return nil, err
}
```